modules/nw-throughput-troubleshoot.adoc

```diff
@@ -1,50 +1,39 @@
+* Use a packet analyzer, such as ping or tcpdump to analyze traffic between a pod and its node.
 +
-For example, run the tcpdump tool on each pod while reproducing the behavior
-that led to the issue.
-Review the captures on both sides to compare send and receive timestamps to
-analyze the latency of traffic to and from a pod.
-Latency can occur in {product-title} if a node interface is overloaded with
-traffic from other pods, storage devices, or the data plane.
+For example, link:https://access.redhat.com/solutions/4569211[run the tcpdump tool on each pod] while reproducing the behavior that led to the issue. Review the captures on both sides to compare send and receive timestamps to analyze the latency of traffic to and from a pod. Latency can occur in {product-title} if a node interface is overloaded with traffic from other pods, storage devices, or the data plane.
 +
 [source,terminal]
 ----
 $ tcpdump -s 0 -i any -w /tmp/dump.pcap host <podip 1> && host <podip 2> <1>
 ----
 +
-<1> `podip` is the IP address for the pod. Run the `oc get pod <pod_name> -o wide` command to get
-the IP address of a pod.
+<1> `podip` is the IP address for the pod. Run the `oc get pod <pod_name> -o wide` command to get the IP address of a pod.
 +
-tcpdump generates a file at `/tmp/dump.pcap` containing all traffic between
-these two pods. Ideally, run the analyzer shortly
-before the issue is reproduced and stop the analyzer shortly after the issue
-is finished reproducing to minimize the size of the file.
-You can also run a packet analyzer between the nodes (eliminating the SDN from
-  the equation) with:
+tcpdump generates a file at `/tmp/dump.pcap` containing all traffic between these two pods. Ideally, run the analyzer shortly before the issue is reproduced and stop the analyzer shortly after the issue is finished reproducing to minimize the size of the file. You can also link:https://access.redhat.com/solutions/5074041[run a packet analyzer between the nodes] (eliminating the SDN from the equation) with:
 +
 [source,terminal]
 ----
 $ tcpdump -s 0 -i any -w /tmp/dump.pcap port 4789
 ----
 
-* Use a bandwidth measuring tool, such as iperf, to measure streaming throughput
-and UDP throughput. Run the tool from the pods first, then from the nodes,
-to locate any bottlenecks.
+* Use a bandwidth measuring tool, such as link:https://access.redhat.com/solutions/6129701[iperf], to measure streaming throughput and UDP throughput. Run the tool from the pods first, then from the nodes, to locate any bottlenecks.
 
 ifdef::openshift-enterprise,openshift-webscale[]
 ** For information on installing and using iperf, see this link:https://access.redhat.com/solutions/33103[Red Hat Solution].
 endif::openshift-enterprise,openshift-webscale[]
+
+* In some cases, the cluster may mark the node with the router pod as unhealthy due to latency issues. Use worker latency profiles to adjust the frequency that the cluster waits for a status update from the node before taking action.
```

Comment:
Could we expand this with a little bit of starter information how to use a worker latency profile like the steps above? Again, this could be a short procedure. If digital rot and procedure change is a concern, we could reference customers to a place in the docs where this is described.

Reply:
CIT suggestion ^

Reply:
yeah for sure. The additional resources section in the assembly links to that information, but I agree a short procedure on how to enable worker profiles would be really useful here